basics/action_system.md

```diff
@@ -113,41 +113,63 @@ Subgroups of a group can form submenus of a menu.
 | :-----: | :------------: | :----------------: | :----------------: | :---------------------: |
-|    F    |       T        |         T          |        T           |         F            |
-|    F    |       T        |         F          |        F           |         x            |
-| ***F*** |       F        |         T          |     ***T***        |      ***T***         |
-|    F    |       F        |         F          |        F           |         x            |
-|    T    |       T        |         T          |        T           |         F            |
-|    T    |       T        |         F          |        F           |         x            |
-| ***T*** |       F        |         T          |     ***F***        |         x            |
-|    T    |       F        |         F          |        F           |         x            |
+|    T    |     **F**      |         T          |      **F**         |         N/A             |
+|    T    |       T        |         T          |        T           |         F               |
+|    F    |     **F**      |         T          |      **T**         |       **T**             |
+|    F    |       T        |         T          |        T           |         F               |
+
+All other combinations of `compact`, visibility, and enablement produce N/A for gray appearance because the menu item isn't visible.
 
 See the [Grouping Actions](/tutorials/action_system/grouping_action.md) tutorial for examples of creating action groups.
 
 ## Registering Actions
 There are two main ways to register an action: either by listing it in the `<actions>` section of a plugin's `plugin.xml` file, or through code.
 
 ### Registering Actions in plugin.xml
-Registering actions in `plugin.xml` is demonstrated in the following example, which documents all elements and attributes that can be used in the `<actions>` section, and describes the meaning of each element.
-This information can also be found by using the [Code Completion](https://www.jetbrains.com/help/idea/auto-completing-code.html#invoke-basic-completion), [Quick Definition](https://www.jetbrains.com/help/idea/viewing-reference-information.html#view-definition-symbols) and [Quick Documentation](https://www.jetbrains.com/help/idea/viewing-reference-information.html#inline-quick-documentation) features.
- 
+Registering actions in `plugin.xml` is demonstrated in the following reference examples, which document all elements and attributes that can be used in the `<actions>` section, and describes the meaning of each element.
+
+#### Setting the Override-Text Element for an Action
+
+> **Tip** Beginning in 2020.1, an alternate version of an action's menu text can be declared for use depending on where an action appears.
+
+By using the `override-text` element introduced in v2020.1 of the IntelliJ Platform, the menu text for an action can be different depending on context: menu location, toolbar, etc.
```

Comment:
IIRC we use `IntelliJ Platform <version>`, where `version` is simply `2020.1` without `v`-prefix